composer.json

```diff
@@ -12,16 +12,16 @@
     },
     "require": {
         "php": ">=5.3.9",
-        "friendsofsymfony/elastica-bundle": "3.1.*",
-        "symfony/symfony": "2.8.*",
+        "friendsofsymfony/elastica-bundle": "3.2.*",
+        "symfony/symfony": "3.2.*",
         "doctrine/orm": "^2.4.8",
         "doctrine/doctrine-bundle": "~1.4",
         "symfony/swiftmailer-bundle": "~2.3",
         "symfony/monolog-bundle": "@stable",
         "sensio/distribution-bundle": "@stable",
         "sensio/framework-extra-bundle": "@stable",
         "incenteev/composer-parameter-handler": "@stable",
-        "hwi/oauth-bundle": "^0.4.0",
+        "hwi/oauth-bundle": "0.5.*-dev",
```

Comment:
I remember having auth problem with the version above 0.4, but maybe I'm wrong.
Did you try to login with this version of oauth-bundle?